bin/adhoc-scripts/updatePileupObjects.py

```diff
@@ -179,23 +190,70 @@ def main():
     # initialize HTTPS client handler
     handler = HTTPSClientAuthHandler(key, cert)
 
-    with open(opts.fin, 'r', encoding='utf-8') as istream:
+    if opts.transition:
+        addTransitionRecords(handler, opts.url, opts.userDN, logger, opts.dryrun)
+    else:
+        updatePileupRecords(handler, opts.url, opts.fin, logger, opts.dryrun)
+
+
+def addTransitionRecords(handler, url, userDN, logger, dryrun):
+    """
+    Helper function to add and update in place transition record for existing MSPileup records
+    :param handler: HTTP handler to use
+    :param url: url to use (string)
+    :param userDN: string representing user DN
+    :param logger: logger to use
+    :param dryrun: option to run dry-run mode (boolean)
+    :return: nothing
+    """
+    puDocs = getPileupDocs(url, handler, logger)
+    logger.info("Found %d documents in MSPileup", len(puDocs))
+    for rec in puDocs:
+        if 'pileupName' not in rec:
```

Comment:
is this case even possible? if not, then please just drop it.

Reply:
this is (was) useful for local testing, but in cmsweb and cmsweb-testbed of course it is not necessary.